src/meta/src/hummock/hummock_manager.rs

```diff
@@ -616,48 +609,6 @@ where
                         .fold(max_committed_epoch, std::cmp::min)
                 };
 
-                // to get all relational table_id from sst_info
-                let table_ids = compact_task
-                    .input_ssts
-                    .iter()
-                    .flat_map(|level| {
-                        level
-                            .table_infos
-                            .iter()
-                            .flat_map(|sst_info| {
-                                sst_info.vnode_bitmaps.iter().map(|bitmap| bitmap.table_id)
-                            })
-                            .collect_vec()
-                    })
-                    .collect::<HashSet<u32>>();
-
-                if compact_task.target_level != 0 {
-                    compact_task.vnode_mappings.reserve_exact(table_ids.len());
-                }
-
-                for table_id in table_ids {
-                    if compact_task.target_level != 0 {
-                        if let Some(vnode_mapping) = self
-                            .env
-                            .hash_mapping_manager()
-                            .get_table_hash_mapping(&table_id)
-                        {
-                            let (original_indices, compressed_data) = compress_data(&vnode_mapping);
-                            let compressed_mapping = ParallelUnitMapping {
-                                table_id,
-                                original_indices,
-                                data: compressed_data,
-                            };
-                            compact_task.vnode_mappings.push(compressed_mapping);
-                        }
-                    }
-
-                    // to found exist table_id from
-                    if existing_table_ids_from_meta.contains(&table_id) {
-                        compact_task.existing_table_ids.push(table_id);
```

Comment:
I think existing_table_ids is required by compaction filter. So it shouldn't be removed. @Li0k

Reply:
> I think existing_table_ids is required by compaction filter. So it shouldn't be removed. @Li0k

fixed